rnc2rng/parser.py

```diff
@@ -473,7 +479,7 @@ def id_kw_div(s, p):
     return Node('NAME', p[0].value)
 
 @pg.production('id-or-kw : INCLUDE')
-def id_kw_div(s, p):
+def id_kw_inc(s, p):
```

Comment:
Would you mind changing this to just `id_kw_include`, instead of `_inc`? I generally don't parse inc as an abbrevation of include, but more like increment.